.circleci/config.yml

```diff
@@ -451,6 +461,10 @@ jobs:
           command: |
               export PATH=$HOME/miniconda/bin:$PATH
               . activate habitat;
+              cd habitat-sim
+              #Rebuild headless
+              python setup.py install --headless --with-bullet --with-cuda
```

Comment:
Does this mean we will only run tests on Hab-Lab with Bullet enabled? We run with/without for sim, should we do the same with lab?

Reply:
^ Currently yes, actually before the PYTHONPATH fix in this PR, we were running ALL our Python tests without bullet including the habitat-sim ones...  0_0

Reply:
🤔  I don't think that is true. 
I ran a quick test to validate this: https://github.com/facebookresearch/habitat-sim/tree/test-bullet-ci-testing

Here, I added an assertion inside a physics test section which is skipped for non-bullet builds and it failed CI.

Reply:
Specifically for pytest